backend/apid/routers/namespaces.go

```diff
@@ -30,8 +48,72 @@ func (r *NamespacesRouter) Mount(parent *mux.Router) {
 	}
 
 	routes.Del(r.handlers.DeleteResource)
-	routes.Get(r.handlers.GetResource)
+	routes.Get(r.get)
 	routes.List(r.handlers.ListResources, corev2.NamespaceFields)
 	routes.Post(r.handlers.CreateResource)
 	routes.Put(r.handlers.CreateOrUpdateResource)
 }
+
+func (r *NamespacesRouter) get(req *http.Request) (interface{}, error) {
+	attrs := authorization.GetAttributes(req.Context())
```

Comment:
We will probably need to set `attrs.Namespace` to `*` or something like this otherwise it will fail [here](https://github.com/sensu/sensu-go/blob/66ab843069daaca27301b9378f8718f43e47d453/backend/authorization/rbac/rbac.go#L75)

Reply:
I've modified VisitRulesFor to add a special case for the namespace resource, so that the namespace can be empty.